package.json

```diff
@@ -16,6 +16,7 @@
   "dependencies": {
     "@codemirror/lang-python": "6.1.0",
     "codemirror": "6.0.1",
+    "electron-forge": "^5.2.4",
```

Comment:
This doesn't need to be added here, since it's already in devDependencies

Reply:
Done.